protocol/testing/e2e/gov/prices_test.go

```diff
@@ -0,0 +1,200 @@
+package gov_test
+
+import (
+	"testing"
+
+	"github.com/cometbft/cometbft/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govtypesv1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+	"github.com/dydxprotocol/v4-chain/protocol/lib"
+	testapp "github.com/dydxprotocol/v4-chain/protocol/testutil/app"
+	"github.com/dydxprotocol/v4-chain/protocol/testutil/constants"
+	pricestest "github.com/dydxprotocol/v4-chain/protocol/testutil/prices"
+	clobtypes "github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
+	perptypes "github.com/dydxprotocol/v4-chain/protocol/x/perpetuals/types"
+	pricestypes "github.com/dydxprotocol/v4-chain/protocol/x/prices/types"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	GENESIS_MARKET_PARAM = pricestypes.MarketParam{
+		Id:                 0,
+		Pair:               "btc-adv4tnt",
+		Exponent:           -8,
+		MinExchanges:       2,
+		MinPriceChangePpm:  1_000,
+		ExchangeConfigJson: "{}",
+	}
+
+	MODIFIED_MARKET_PARAM = pricestypes.MarketParam{
+		Id:                 0,
+		Pair:               "eth-adv4tnt",
+		Exponent:           -8,
```

Comment:
nit: you can use `GENESIS_MARKET_PARAM.Exponent` here instead of -8 with a comment that exponents cannot change.

Reply:
... you could also use `GENESIS_MARKET_PARAM.Id` to populate the `Id` field.